hack/rosa-cleanup.py

```diff
@@ -82,49 +76,24 @@ def prune_load_balancers(clusters, resources):
     client = boto3.client('elbv2')
     arns = []
     for lb in resources['loadBalancerTags']:
-        for tag in lb['Tags']:
-            found = False
-            if tag['Key'] == 'api.openshift.com/id':
-                for cluster in clusters:
-                    if tag["Value"] == cluster['id']:
-                        found = True
-                        break
-                if not found:
-                    arns.append(lb["ResourceArn"])
+        if not associated_with_active_cluster(lb['Tags'], clusters):
```

Comment:
The logic here is the not the same as what `associated_with_active_cluster` replaces. But if the new logic in `associated_with_active_cluster` covers what you want, this is ok.

Reply:
AFAICT, other than checking for a couple different tags at once, the logic should be pretty much identical